app/views/posts/show.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Posts#show</h1>
-<h2>This is the posts show page</h2>
\ No newline at end of file
+<div class="main-container">
+	<div class="post-container">
+		<h2><%= @post.title %> by <%= @post.author.name %></h2>
+		<h6>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></h6>
+		<p class="post-text"><%= @post.text %></p>
+	</div>
+
+	<div class="comments-cont">
+		<% if @post.all_comments.count.zero? %>
+		<p>Post has no comments yet</p>
+		<%else%>
+		<% @post.all_comments.each do |comment| %>
+			<p><%= comment.author.name %>: <%= comment.text %></p>
+		<% end %>
+		<% end %>
+	</div>
+</div>
```

Comment:
- [ Required ] Good job implementing some changes from the previous reviewer. Since some parts of the app weren't working during the first review, I think it would be better if you made sure that the `posts` show page works as expected. Currently, when I open the `posts` show page I get the error below. Please consider fixing this error.

## Screenshot
![Screenshot from 2023-06-16 01-59-52](https://github.com/Abe1able/Bloggingly/assets/16392046/dcfa6d21-502b-4d12-bd4c-71f69a7915b4)

Reply:
- [ ] just as the previous reviewer mentioned when I navigate to you post show page, it throws an error, kindly fix this
![Screenshot from 2023-06-17 00-15-09](https://github.com/Abe1able/Bloggingly/assets/66097371/f3659f83-3e5a-42b2-81f4-befa90b2c889)